Automattic/wp-calypso
# Add Theme Setup to the `wpcalypso` environment.

This will be the final step of adding Theme Setup #10542 to `wpcalypso` for a8c-only testing. To be merged only after #11812 .

**Testing**

* Switch to this PR and go to https://calypso.local:3000/settings/general/your-dotcom-test-domain
* Make sure there's an option for Theme Setup (then click through to be sure you see the Theme Setup card):

![screen shot 2017-03-09 at 4 08 47 pm](https://cloud.githubusercontent.com/assets/349751/23776584/fcae1758-04e3-11e7-8811-b50a770eb944.png)

![screen shot 2017-03-09 at 4 26 25 pm](https://cloud.githubusercontent.com/assets/349751/23776782/3452a65a-04e5-11e7-957c-91e70e3d8559.png)

* Switch to a Jetpack site; the direct URL (`settings/theme-setup/site-domain`) should forward you back to `settings/general`, and there should be no Theme Setup choice now:

![screen shot 2017-03-09 at 4 12 24 pm](https://cloud.githubusercontent.com/assets/349751/23776729/e256e410-04e4-11e7-9d08-d9e1707266ec.png)

* Switch back to a WordPress.com site.
* Change `settings/theme-setup` to something invalid, to mimic unsupported environments. Change `client/my-sites/site-settings/delete-site-options/index.jsx` L78, and `client/my-sites/site-settings/controller.js` L199. Theme Setup should not be available.
